pkg/virt-handler/vm.go

```diff
@@ -1943,8 +1943,8 @@ func lookupVMIPodNetworkName(networks []v1.Network) string {
 
 func (d *VirtualMachineController) validateSRIOVInterfacesForMigration(vmi *v1.VirtualMachineInstance) error {
 	for _, iface := range vmi.Spec.Domain.Devices.Interfaces {
-		if iface.SRIOV != nil && !d.clusterConfig.SRIOVLiveMigrationEnabled() {
-			return fmt.Errorf("SRIOVLiveMigration feature-gate is closed, can't migrate VMI with SRIOV interfaces")
+		if iface.SRIOV != nil {
```

Comment:
I think it is very confusing to have this `SRIOVLiveMigrationEnabled` feature gate but ignoring it.
Moreover, it is not completely ignored since it affects the `LiveMigrationEnabled` feature gate.

In this stage, I would not allow enabling this feature gate.

Reply:
We need to have it in order to allow on one hand to test it opens up the capability and on the other still refusing to migrate.
The PR after this, will handle the source migration part (un-plug) where we can open this up and show through tests that migration works (with the limitation that on the target there is no plugin action done).

Reply:
Let's at least have a different error message for each case.
If the `SRIOVLiveMigrationEnabled` is `false` we can keep the "virtual machine with SRIOV interfaces live migration is not supported" error message.
`SRIOVLiveMigrationEnabled` is `true`, I would add a different error message specifying the although `SRIOVLiveMigrationEnabled` is `true`, the migration is still not supported.

Reply:
Done